packages/manager/modules/bm-server-components/src/ipmi/index.js

```diff
@@ -0,0 +1,28 @@
+import angular from 'angular';
+
+import '@ovh-ux/ui-kit';
+import 'angular-translate';
+import '@ovh-ux/ng-translate-async-loader';
+import '@ovh-ux/ng-at-internet';
+import ovhManagerCore from '@ovh-ux/manager-core';
+
+import component from './component';
+import service from './service';
+import polling from '../polling/polling.service';
+
+const moduleName = 'ovhManagerBmServerComponentsIpmiComponent';
+
+angular
+  .module(moduleName, [
+    'oui',
+    'pascalprecht.translate',
+    'ngTranslateAsyncLoader',
+    'ngAtInternet',
+    ovhManagerCore,
+  ])
+  .component('serverIpmi', component)
+  .service('IpmiService', service)
+  .service('Polling', polling)
```

Comment:
beware that polling is also used here for example :

https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/dedicated/server/details/server.service.js#L27

if you only define the service in the lazy loaded ipmi module it won't be available for the rest of manager dedicated until that module is loaded

Reply:
I have moved polling into this module and its been injected in dedicated app at root level.